lib/anyway/config.rb

```diff
@@ -155,5 +190,24 @@ def parse_yml(path)
         YAML.load_file(path)
       end
     end
+
+    def build_option_parser
```

Comment:
What about extracting this functionality into its own module?
For example, `OptionParserBuilder.call(options)` where `options` is a Hash of options names as keys and descriptions (or `nil`-s) as values?

Then we only need to generate a proper `options` Hash in the config itself.

That would make testing a little bit easier.

Reply:
Sounds good